src/mlpack/methods/amf/init_rules/average_init.hpp

```diff
@@ -30,15 +30,15 @@ class AverageInitialization
   // Empty constructor required for the InitializeRule template
   AverageInitialization() { }
 
-  /**
-   * Initialize the matrices W and H to the average value of V with uniform
-   * random noise added.
-   *
-   * @param V Input matrix.
-   * @param r Rank of matrix.
-   * @param W W matrix, to be initialized.
-   * @param H H matrix, to be initialized.
-   */
+   /**
+    * Initialize the matrices W and H to the average value of V with uniform
+    * random noise added.
+    *
+    * @param V Input matrix.
+    * @param r Rank of matrix.
+    * @param W W matrix, to be initialized.
+    * @param H H matrix, to be initialized.
+    */
```

Comment:
This looks like it's indented one space too many?

Reply:
Hi @rcurtin, I add one more space here based on the comment from @zoq above. Now I'm not sure if there should be 3 spaces or 4 spaces here. I believe there are only 3 spaces here in old branches.

Reply:
`/**` was right from the the start, only the following lines had to be adjusted.

Reply:
@zoq Do you mean that all the following lines shall aligned to the second asterisk in `/**`? Or in other words, `/**` has 2 leading spaces and the following lines have 4 leading spaces?